src/orderstationV1/OrderStationSmokeTestSuite.java

```diff
@@ -0,0 +1,217 @@
+package orderstationV1;
```

Comment:
The test steps are not readable, you need to read about page object model in writing your tests